gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
```

Comment:
I'm guessing we only need the binary files so that's why we're grabbing the `bin` file now instead of the `all` file?

Reply:
This was generated by the `./gradlew wrapper` command I used to update the gradle version. It chooses what to download.

Reply:
cool! didn't know about that command